flopy/modflow/mfhob.py

```diff
@@ -677,22 +679,18 @@ def __init__(
                     "{} names are required.".format(len(names), self.nobs)
                 )
 
-        # set use_cached_totim to False first to ensure totim is updated
-        use_cached_totim = False
-
         # create time_series_data
         self.time_series_data = self._get_empty(ncells=shape[0])
         for idx in range(self.nobs):
             t = time_series_data[idx, 0]
             kstp, kper, toffset = model.dis.get_kstp_kper_toffset(
-                t, use_cached_totim
+                t, use_cached_totim = True
```

Comment:
When specifying an argument value in the constructor, there should not be spaces before and after the equals sign.  If you fix this and resubmit, you may not need to run black.